README.md

```diff
@@ -233,10 +226,13 @@ You are now ready to start developing your own functions. Please remember to fol
 > **_NOTE:_**  Ensure you have read the [Release Guidelines](RELEASE.md) before publishing your code.
 
 # Support and contacts
+
 If you encounter any issues or have questions, please reach out to the Innersource-Enablement-Team. They will be happy to assist you and provide further information about the project.
```

Comment:
I suspect the Innersource-Enablement-Team is an internal reference that should be removed?

Reply:
We've fixed this in our latest pr to our main branch so people are directed to raise issues to reach out to us.